packages/training/deploy/docker/Dockerfile.base

```
@@ -0,0 +1,72 @@
+    postgresql-client \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install Bun for database migrations
+RUN curl -fsSL https://bun.sh/install | bash
+ENV PATH="/root/.bun/bin:$PATH"
+
+# Create app directory
+WORKDIR /app
+
+# Install additional ML dependencies for training
+RUN pip install --no-cache-dir \
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)